.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Build dist and publish to PyPI
+
+on: [push, pull_request]
```

Comment:
I don't think we want this triggered on `pull_request`.  In the end / for main, I think this should really be just for `release`.

Reply:
That's fair! I'll go ahead and fix that